### PR TITLE
ACI UTIL: url building redesign moving items to module instead of util

### PR DESCRIPTION
**Pull request by jmcgill298**
…instead of util

### CODE REVIEW

**Comment by jmcgill298** [comment]
@dagwieers I have sampled a few modules for URL redesign; let me know if you prefer this design and I can convert the other modules. If you don't like it, then I will can this branch.

BTW, this did pass the integration tests I wrote for `aci_bd_subnet`

File: library/aci_config_snapshot.py [hunk 1]
```diff
@@ -143,17 +143,24 @@ def main():
         if max_count in range(1, 11):
             max_count = str(max_count)
         else:
-            module.fail_json(msg='The "max_count" must be a number between 1 and 10')
+            module.fail_json(msg='The C(max_count) must be a number between 1 and 10')
```
Comment (dagwieers) [inline library/aci_config_snapshot.py]: I don't think we can use these markers in python output :-)

Comment (jmcgill298) [reply library/aci_config_snapshot.py]: ya, that makes sense. I will update, and continue converting modules and writing tests

**Comment by jmcgill298** [comment]
@dagwieers I have fixed your requests, converted all modules, and added integration tests for about 1/2 of them.

**Comment by dagwieers** [comment]
@jmcgill298 I hope you like it too ;-) LGTM !

**Comment by jmcgill298** [comment]
@dagwieers it is definitely a cleaner interface

**Pull request merged**